controllers/sriovoperatorconfig_controller.go

```diff
@@ -61,6 +61,7 @@ func (r *SriovOperatorConfigReconciler) Reconcile(req ctrl.Request) (ctrl.Result
 
 	logger.Info("Reconciling SriovOperatorConfig")
 
+	enableAdmissionController := os.Getenv("ENABLE_ADMISSION_CONTROLLER") == "true"
```

Comment:
Please add a info log here to indicate admission controller is disabled.

Reply:
updated